### PR TITLE
Move examples from src/lib to src/examples

### PR DESCRIPTION
**Pull request by imobachgs**
IMHO, that example code belongs to `src/examples`.

### CODE REVIEW

File: src/examples/used_storage_features_demo.rb [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+# with this program; if not, contact SUSE LLC.
+#
+# To contact SUSE LLC about this file by physical or electronic mail, you may
+# find current contact information at www.suse.com.
+
+$LOAD_PATH.unshift(File.expand_path("../../lib", __FILE__))
+
+require "y2storage"
+
+devicegraph = Y2Storage::StorageManager.instance.probed
```
Comment (ancorgs) [inline src/examples/used_storage_features_demo.rb]: We should probably show a warning if not executed as root

Comment (shundhammer) [inline src/examples/used_storage_features_demo.rb]: Oh yes, right...

Comment (shundhammer) [inline src/examples/used_storage_features_demo.rb]: Now really LGTM.

**Comment by imobachgs** [comment]
Thanks!

**Comment by ancorgs** [comment]
LGTM also :)

**Pull request merged**